subworkflows/nf-neuro/bundle_seg/meta.yml

```diff
@@ -39,13 +42,7 @@ args:
       default: ""
       description: |
         Use an alternative atlas to the default BundleSeg one available on Zenodo (https://zenodo.org/records/10103446).
-        The folder MUST follow this specific structure:
-              atlas_directory
-              ├── atlas
-              │   └── pop_average
-              ├── centroids
-              ├── *.json          (config file)
-              └── *.{nii,nii.gz}  (atlas anatomical file)
+        The folder MUST follow the same organisation as the default one.
```

Comment:
Unless it's horrible in the documentation I would keep the tree. I don't understand the default one 😄

Reply:
We can't break lines cleanly in a **markdown table**, everything is contiguous, sadly 😅 Code components don't work either. I was able to manage the `Structure :` part of the input/output description, but I found nothing to make this one work.

Maybe a `FileTree` astro component would work, but it would be super ugly when not rendered to the website (like in the CLI).

Reply:
Could we have a documentation page somewhere in the SCIL/sclpy documentation and link to it ?